src/env.rs

```diff
@@ -32,12 +32,15 @@ impl WasmerEnv for Env {
     }
 }
 
-pub fn get_remaining_points_for_env(env: &Env) -> u64 {
-    let instance = &env.instance.clone().unwrap();
-    match metering::get_remaining_points(instance) {
+pub fn get_remaining_points_for_env(env: &Env) -> ABIResult<u64> {
+    let instance = match env.instance.clone() {
```

Comment:
How do you know it's that ?

Reply:
I saw the first line of the error in the associated issue,
```
thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', /home/labnet/.cargo/git/checkouts/massa-sc-runtime-7b4aa98a779b277a/32b3beb/src/env.rs:51:42
```

I profit of the current commit to eliminate arbitrary unwraps. Now the error is thrown since the line here in `execution_impl.rs`
```rust
    Ok(Instance::new(&module, &resolver)?)
```